docs/index.rst

```diff
@@ -409,7 +408,7 @@ Relevant configuration values
 - CACHE_MEMCACHED_SERVERS
 
 
-.. note:: Werkzeug does not pass additional configuration options
+.. note:: Flask-Cached does not pass additional configuration options
```

Comment:
Flask-Caching

Reply:
I wear the cone of shame! I'll fix it now.